source/includes/administration/_organizations.md

```diff
@@ -30,12 +30,20 @@ curl "https://cloudmc_endpoint/api/v1/organizations" \
          "isReseller": false,
          "tags": [
             {
-              "id": "78c6afa5-8b65-40a5-be1c-aa55a3b81030",
-              "name": "a-tag",
-              "organization": {
-                "id":"075d64d5-c629-481a-9b91-32e5e286aeae",
-              },
+               "system": true,
+               "name": "billable",
+               "id": "5a84485f-7eed-11ec-b819-0242ac120002"
```

Comment:
We do not have the reseller id anymore? I guess it is not needed.

Reply:
yeah its not really relevant